lua/lazyvim/plugins/extras/editor/fzf.lua

```diff
@@ -0,0 +1,157 @@
+local M = {}
+
+---@class FzfLuaOpts: table<string, any>
+---@field cwd boolean|string
+
+---@param opts? {cwd?:string|boolean}
+function M.fzf(builtin, opts)
+  opts = opts or {}
+
+  return function()
+    local dir = opts.cwd == true and vim.uv.cwd() or LazyVim.root()
+    opts.cwd = dir
+    if
+      builtin == "files"
+      and vim.uv.fs_stat(dir .. "/.git")
+      and not vim.uv.fs_stat(dir .. "/.ignore")
+      and not vim.uv.fs_stat(dir .. "/.rgignore")
+    then
+      if opts.cmd == nil then
+        opts.cmd = "git ls-files --exclude-standard --cached --others"
+      end
+      builtin = "git_files"
+    end
+
+    return require("fzf-lua")[builtin](opts)
+  end
+end
+
+return {
+  {
+    "nvim-telescope/telescope.nvim",
+    enabled = true,
+    keys = function()
+      return {}
+    end,
+  },
+  {
+    "ibhagwan/fzf-lua",
+    event = "VeryLazy",
+    opts = {
+      [1] = "default-title",
+      keymap = {
+        fzf = {
+          ["ctrl-q"] = "select-all+accept",
+        },
+      },
```

Comment:
@folke Just a suggestion. By doing that you overwrite all the default keymaps as also mentioned in the [docs](https://github.com/ibhagwan/fzf-lua/blob/9ecc8f6a6555cd3988c21480069aa5da001cd9df/README.md?plain=1#L606-L609). If you don't intend to do this, then take a look at my [dots](https://github.com/dpetka2001/dotfiles/blob/a95a1cb3650c7e8a9f138b2067d1f4a544d7de88/dot_config/nvim/lua/plugins/editor.lua#L170-L200) for alternatives how to not completely overwrite them.

Reply:
Yeah, I actually know. Forgot that was still in there. Should be config.globals or config.defaults.

But I'll definitely check your dots!

I also want that bug I referenced fixed. Something odd is going on there. Or its just my limited understanding